x/posts/internal/types/post_response.go

```diff
@@ -7,13 +7,34 @@ import (
 // PostQueryResponse represents the data of a post
 // that is returned to user upon a query
 type PostQueryResponse struct {
-	Post
+	Type      string    `json:"type"`
+	Post      Post      `json:"post"`
 	Reactions Reactions `json:"reactions"`
 	Children  PostIDs   `json:"children"`
 }
 
+func (response PostQueryResponse) String() string {
```

Comment:
Why is this method necessary?

Reply:
to make a correct query output in `query.go` at line 55 	
```golang
return cliCtx.PrintOutput(out)
 ```